flocker/node/test/test_loop.py

```diff
@@ -724,6 +726,43 @@ def test_status_update_while_sleeping_no_discovery(self):
             remaining_discover_calls - len(self.deployer.local_states),
             0)
 
+    def test_longer_sleep_when_converged(self):
+        """
+        When a convergence loop results in a ``NoOp`` the sleep is based on
+        that configured on the ``Deployer``.
+        """
+        loop = self.convergence_iteration(initial_action=NoOp(),
+                                          later_actions=[NoOp(), NoOp()])
+
+        # Another update received while sleeping:
```

Comment:
`convergence_iteration()` involves an update, so this is the second one. But that is a little unclear, I'll change it.